### DISCUSSION TITLE
input radiation unit

### DISCUSSION THREAD

**Discussion by wxystudio** [APSIMInitiative/ApsimX#8794] [General]
I'm using the APSIM crop model to simulate maize yield. The input features include the "Radiation", whose unit is MJ/m^2.

I want to use some radiation datasets from Google earth engine as input to the crop model, and I find the MCD18A1.061 Surface Radiation Daily/3-Hour dataset [link](https://developers.google.com/earth-engine/datasets/catalog/MODIS_061_MCD18A1#bands)

But the unit of this dataset is W/m^2, could you tell me how to transfer it to MJ/m^2, or where can I find a radiation dataset with MJ/m^2 unit on GEE?

I try to use this equation to convert the value: 

![image](https://github.com/APSIMInitiative/ApsimX/assets/17384283/401fc34f-c4ac-428b-a534-8044bd29e366)

But the transferred value range is a little weird. For example, the radiation value used in crop models is generally around 10, and after converting it to W/m^2 using this formula, it becomes 2777 W/m^2. However, this value is obviously much higher than the max value in the GEE dataset, which is 1400.

**Comment by zur003** [comment]
You need to consider time in your conversion, since Watts and MegaJoules measure different things. Megajoules describe an _amount_ of energy, whereas watts describe the _rate__ at which energy is being transferred. The formula you link to shows the conversion from 1 Watt-HOUR to MJ. 
In a way, the units used in APSIM are not quite accurate - it's actually MJ/m^2/_day_ - that is, the total radiant energy received over the course of a day. The GGE values are, I think, instantaneous values for the rate of radiant energy input. Converting between them can be tricky, as the rate of energy input will obviously change over the course of the day. You can see, though, that your application of the formula yielded 2777 W**h**/m^2, and if we (simplistically) assume that radiation was received over 10 hours of daylight, the rate of radiation averages to a much more realistic 277.7 W/m^2

**Reply by wxystudio** [reply]
Thanks! Very insightful idea.
